content/en/docs/started/introduction.md

```diff
@@ -4,11 +4,43 @@ description = "An introduction to Kubeflow"
 weight = 1
 +++
 
-The Kubeflow project is dedicated to making deployments of machine learning (ML)
-workflows on Kubernetes simple, portable and scalable. Our goal is not to
-recreate other services, but to provide a straightforward way to deploy
-best-of-breed open-source systems for ML to diverse infrastructures. Anywhere
-you are running Kubernetes, you should be able to run Kubeflow.
+## What is Kubeflow ?
+
+Kubeflow is a community and ecosystem of open-source projects to address each stage in the
+machine learning (ML) lifecycle. It makes ML on Kubernetes simple, portable, and scalable.
+Its goal is to facilitate the orchestration of Kubernetes workloads for ML and to empower users to
```

Comment:
```suggestion
The goal is to facilitate the orchestration of Kubernetes ML workloads and to empower users to
```

Reply:
Does it sound good @StefanoFioravanzo ?
The Kubeflow goal is to facilitate the orchestration of Kubernetes ML workloads and to empower users
to deploy best-in-class open-source systems for ML to diverse cloud infrastructures.

Reply:
Just rephrase the first part: `The goals of Kubeflow` or just `The goal`, otherwise it sounds weird.